input/pagecontent/terminology.md

```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
+|[Australian Pronouns](https://healthterminologies.gov.au/fhir/ValueSet/australian-pronouns-1)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|NCTS|
+|[Australian States and Territories](https://healthterminologies.gov.au/fhir/ValueSet/australian-states-territories-2)|[Australian Address](StructureDefinition-au-address.html)|NCTS|
+|[Australian Veteran Status](https://healthterminologies.gov.au/fhir/ValueSet/australian-veteran-status-1)|[Australian Veteran Status](StructureDefinition-veteran-status.html)|NCTS|
+|[Body Site](https://healthterminologies.gov.au/fhir/ValueSet/body-site-1)|[AU Base Body Structure](StructureDefinition-au-bodystructure.html), [AU Base Condition](StructureDefinition-au-condition.html), [AU Base Diagnostic Imaging Result](StructureDefinition-au-imagingresult.html), [AU Base Diagnostic Request](StructureDefinition-au-diagnosticrequest.html), [AU Base Dosage](StructureDefinition-au-dosage.html), [AU Base Medication Administration](StructureDefinition-au-medicationadministration.html), [AU Base Procedure](StructureDefinition-au-procedure.html), [AU Base Specimen](StructureDefinition-au-specimen.html)|NCTS|
+|[Body Site Relative Site Qualifier](https://healthterminologies.gov.au/fhir/ValueSet/body-site-relative-site-qualifier-1)|[AU Base Body Structure](StructureDefinition-au-bodystructure.html)|NCTS|
+|[Clinical Condition](https://healthterminologies.gov.au/fhir/ValueSet/clinical-condition-1)|[AU Base Condition](StructureDefinition-au-condition.html)|NCTS|
+|[Clinical Finding](https://healthterminologies.gov.au/fhir/ValueSet/clinical-finding-1)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html), [AU Base Condition](StructureDefinition-au-condition.html), [AU Base Diagnostic Imaging Report](StructureDefinition-au-imagingreport.html), [AU Base Diagnostic Report](StructureDefinition-au-diagnosticreport.html), [AU Base Dosage](StructureDefinition-au-dosage.html), [AU Base Pathology Report](StructureDefinition-au-pathologyreport.html)|NCTS|
+|[Clinical Specialty](https://healthterminologies.gov.au/fhir/ValueSet/clinical-specialty-1)|[AU Base Healthcare Service](StructureDefinition-au-healthcareservice.html), [AU Base Practitioner Role](StructureDefinition-au-practitionerrole.html)|NCTS|
+|[Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-jurisdiction.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
```

Comment:
These Value Set names have changed to remove Common. Please update.

Reply:
Updated to remove common, thanks.

Reply:
Amended all instances, thanks.